config/application.rb

```diff
@@ -146,6 +146,12 @@ class Application < Rails::Application
         child.helper helpers
       end
     end
+
+    # Secure cookies if the connection is via SSL
+    if SETTINGS[:require_ssl] == true
```

Comment:
My only nitpick is that it's different to https://github.com/theforeman/foreman/pull/2328/files#diff-b4f92b4e361ffab4fbb91c68eb3f77bbR69 in behaviour.